Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/Application.cs

```diff
@@ -4,15 +4,14 @@
 
 	public enum WindowSoftInputModeAdjust
 	{
+		Unspecified,
 		Pan,
 		Resize
 	}
 
 	public static class Application
 	{
-		public static readonly BindableProperty WindowSoftInputModeAdjustProperty =
-			BindableProperty.Create("WindowSoftInputModeAdjust", typeof(WindowSoftInputModeAdjust),
-			typeof(Application), WindowSoftInputModeAdjust.Pan);
+		public static readonly BindableProperty WindowSoftInputModeAdjustProperty = BindableProperty.Create(nameof(WindowSoftInputModeAdjust), typeof(WindowSoftInputModeAdjust), typeof(Application), WindowSoftInputModeAdjust.Pan);
```

Comment:
Since `Unspecified` was added, it might make sense to rename this property to `WindowSoftInputMode`. I think a breaking change here should be tolerated, but I haven't made any changes.